.github/workflows/prow-action.yml

```diff
@@ -0,0 +1,20 @@
+# Adds support for prow-like commands
+# Uses .github/labels.yaml to define areas and kinds
+name: "Prow github actions"
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1
+        with:
+          # Only support /kind command for now.
+          # TODO: before allowing the /lgtm command, see if we can block merging if changelog labels are missing.
```

Comment:
I'm wondering if we can use the `/lgtm` command, but make it work *only if* there's either a `has-changelog` (which should be set with the action on this PR) or `changelog-not-required` label on the PR.

I think this way, we could retire the confusing changelog CI check that marks a perfectly-fine PR as failing.

Reply:
/lgtm 😆